backend/src/hatchling/metadata/core.py

```diff
@@ -503,7 +530,7 @@ def readme(self):
             else:
                 raise TypeError('Field `project.readme` must be a string or a table')
 
-            self._readme = self._readme
+            self._readme = self._readme  # TODO: should this be here?
```

Comment:
In a few places `_ = self.PROPERTY` is used to ensure that the side-effects of a property being read have occurred, but that's not the case here. Is it okay to remove this line?